### DISCUSSION TITLE
a command to close the cheat

### DISCUSSION THREAD

**Discussion by Lipti-pl** [winstxnhdw/lc-hax#543] [Ideas]
add a command that just crashes the cheat
would be useful for adjusting it or just turning it off without having to restart the game

**Comment by winstxnhdw** [comment]
I'll have to fix the injector to do this :/ Maybe next time.

**Reply by Lipti-pl** [reply]
> I'll have to fix the injector to do this :/ Maybe next time.

i hope you can because i can see countless situations where a command like that would be useful such as subtle trolling 
i wish you luck :)
